rules/cross-platform/persistence_shell_profile_modification.toml

```
@@ -0,0 +1,79 @@
+[metadata]
+creation_date = "2021/01/19"
+maturity = "production"
+updated_date = "2021/01/19"
+
+[rule]
+author = ["Elastic"]
+description = """
+Both ~/.bash_profile and ~/.bashrc are files containing shell commands that are run when Bash is invoked. These files
+are executed in a user's context, either interactively or non-interactively, when a user logs in so that their
+environment is set correctly. Adversaries may abuse this to establish persistence by executing malicious content
+triggered by a user’s shell.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*", "auditbeat-*"]
+language = "kuery"
+license = "Elastic License"
+name = "Bash Shell Profile Modification"
+references = ["https://www.anomali.com/blog/pulling-linux-rabbit-rabbot-malware-out-of-a-hat"]
+risk_score = 47
+rule_id = "e6c1a552-7776-44ad-ae0f-8746cc07773c"
+severity = "medium"
+tags = ["Elastic", "Host", "macOS", "Linux", "Threat Detection", "Execution", "Persistence"]
+type = "query"
+
+query = '''
+event.category:file and event.action:modification and
+  process.name:
+    (* and not
+      (
+        sudo or
+        vim or
+        zsh or
+        env or
+        nano or
+        bash or
+        Terminal or
+        xpcproxy or
+        login or
+        cat or
+        cp or
+        launchctl or
+        java
+      )
+    ) and
+  not process.executable:(/Applications/* or /private/var/folders/* or /usr/local/*) and
+  file.path:
+    (
+      /private/etc/rc.local or
+      /etc/rc.local or
+      /home/*/.profile or
+      /home/*/.profile1 or
+      /home/*/.bash_profile or
+      /home/*/.bash_profile1 or
+      /home/*/.bashrc or
+      /Users/*/.bash_profile or
+      /Users/*/.zshenv
+    )
```

beautify with more whitespace consideration (due to the whitespace normalization of kql queries) (sorry @bm11100)

Also I changed from `event.action` to `event.type`, unless there was a reason for it @Samirbous 

```suggestion
event.category:file and event.type:change and
  process.name:(* and not (sudo or
                           vim or
                           zsh or
                           env or
                           nano or
                           bash or
                           Terminal or
                           xpcproxy or
                           login or
                           cat or
                           cp or
                           launchctl or
                           java)) and
  not process.executable:(/Applications/* or /private/var/folders/* or /usr/local/*) and
  file.path:(/private/etc/rc.local or
             /etc/rc.local or
             /home/*/.profile or
             /home/*/.profile1 or
             /home/*/.bash_profile or
             /home/*/.bash_profile1 or
             /home/*/.bashrc or
             /Users/*/.bash_profile or
             /Users/*/.zshenv)
```